src/app/calendar/ParcelCalendar.tsx

```diff
@@ -41,18 +43,32 @@ const ParcelCalendar: React.FC<ParcelCalendarProps> = (props) => {
         };
     };
 
-    const colorMap: LocationColorMap = {
-        "Brixton Hill - Methodist Church": getColorText("lightRed"),
-        "Clapham - St Stephens Church": getColorText("lightOrange"),
-        "N&B - Emmanuel Church": getColorText("lightYellow"),
-        "Streatham - Immanuel & St Andrew": getColorText("lightGreen"),
-        "Vauxhall Hope Church": getColorText("darkGreen"),
-        "Waterloo - Oasis": getColorText("lightBlue"),
-        "Waterloo - St George the Martyr": getColorText("darkBlue"),
-        "Waterloo - St Johns": getColorText("lightPurple"),
-        Delivery: getColorText("darkPurple"),
-        default: getColorText("lightBrown"),
-    };
+    // color order deliberately scattered to allow more variety of color in the calendar
```

Comment:
Don't think this comment is necessary ngl

Reply:
Removed